<ui version="4.0" >
 <class>RawFileParserDialog</class>
 <widget class="QDialog" name="RawFileParserDialog" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>369</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Create trace</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="title" >
      <string>Input</string>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>20</y>
        <width>601</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Video File:</string>
         </property>
         <property name="buddy" >
          <cstring>inputFileEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputFileEdit" />
       </item>
       <item>
        <widget class="QPushButton" name="inputFileBrowseButton" >
         <property name="text" >
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="title" >
      <string>Output</string>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <property name="geometry" >
       <rect>
        <x>11</x>
        <y>21</y>
        <width>601</width>
        <height>122</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="outputVideoCheck" >
         <property name="text" >
          <string>Trace video stream</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="outputVideoEdit" />
         </item>
         <item>
          <widget class="QPushButton" name="outputVideoBrowseButton" >
           <property name="text" >
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="outputAudioCheck" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>Trace audio stream</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="outputAudioEdit" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="outputAudioBrowseButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="timeIndexCheck" >
        <property name="text" >
         <string>Export time index</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QProgressBar" name="parseProgressBar" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="value" >
        <number>0</number>
       </property>
       <property name="textVisible" >
        <bool>true</bool>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="parseButton" >
       <property name="minimumSize" >
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Parse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>outputVideoCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>outputVideoEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>89</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel" >
     <x>225</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outputVideoCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>outputVideoBrowseButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>137</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel" >
     <x>571</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outputAudioCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>outputAudioEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>73</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel" >
     <x>94</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outputAudioCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>outputAudioBrowseButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>160</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel" >
     <x>570</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>RawFileParserDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>479</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel" >
     <x>487</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
